### PR TITLE
Dvfs chmod

### PR DESCRIPTION
**Pull request by alexkalmuk**
What's done:
* syscall chmod() is added for both of old and new fs's
* chmod cmd is posix now

It's an open question how to sync an inode's data/metadata with a storage during execution of chmod(), but currently we leave that as is.

### CODE REVIEW

**Comment by anton-bondarev** [comment]
LGTM

File: src/compat/posix/fs/dvfs/chmod.c [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+ *
+ * @date 2.02.2016
+ * @author Alex Kalmuk
+ */
+
+#include <sys/stat.h>
+#include <errno.h>
+#include <fs/dvfs.h>
+
+int chmod(const char *path, mode_t mode) {
```
Comment (Deryugin) [inline src/compat/posix/fs/dvfs/chmod.c]: I think it's better to check correctness of given "mode" arg.
Also I don't see where information is being updated on the media.

Comment (alexkalmuk) [inline src/compat/posix/fs/dvfs/chmod.c]: > Also I don't see where information is being updated on the media. 

This is explained in PR's description - currently there is no sync with external storage. Let's suppose for now to use tmpfs or some similar thing.

**Comment by Deryugin** [comment]
LGTM

**Pull request merged**